tf-modules/archive/process_dead_letter_archive.tf

```diff
@@ -91,12 +91,16 @@ resource "aws_lambda_function" "process_dead_letter_archive" {
 
   environment {
     variables = {
-      system_bucket   = var.system_bucket
-      stackName       = var.prefix
-      ExecutionsTable = var.dynamo_tables.executions.name
-      GranulesTable   = var.dynamo_tables.granules.name
-      PdrsTable       = var.dynamo_tables.pdrs.name
-      dbHeartBeat     = var.rds_connection_heartbeat
+      system_bucket             = var.system_bucket
+      stackName                 = var.prefix
+      ExecutionsTable           = var.dynamo_tables.executions.name
+      GranulesTable             = var.dynamo_tables.granules.name
+      PdrsTable                 = var.dynamo_tables.pdrs.name
+      acquireTimeoutMillis      = var.rds_connection_timing_configuration.acquireTimeoutMillis
+      createRetryIntervalMillis = var.rds_connection_timing_configuration.createRetryIntervalMillis
+      createTimeoutMillis       = var.rds_connection_timing_configuration.createTimeoutMillis
+      idleTimeoutMillis         = var.rds_connection_timing_configuration.idleTimeoutMillis
+      reapIntervalMillis        = var.rds_connection_timing_configuration.reapIntervalMillis
       databaseCredentialSecretArn    = var.rds_user_access_secret_arn
       RDS_DEPLOYMENT_CUMULUS_VERSION = "5.0.0"
```

Comment:
Same note re: env consistency

Reply:
resolving in favor of single threaded discussion